experiments/tester/src/RNTester/TestComponents/Common/AlignmentPicker.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { undefinedText } from '../PersonaCoin/styles';
+import { IconAlignment } from 'react-native-uifabric';
+import { Picker, StyleProp, ViewStyle } from 'react-native';
+
+
+const alignmentValues: Array<typeof undefinedText | IconAlignment> = [undefinedText, 'start', 'center', 'end'];
+
+interface IAlignmentPickerProps {
+  style?: StyleProp<ViewStyle>;
+  label: string;
+  onSelectionChange: (value: IconAlignment | undefined) => void;
+}
+
+export const AlignmentPicker: React.FunctionComponent<IAlignmentPickerProps> = (props: IAlignmentPickerProps) => {

```

Comment:
I'm extracting this component out of `CustomizeUsage.tsx` file below. No code change in this file.